src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -196,6 +204,9 @@ async function startToolSetupFlow(
         configureAngularSourcemapGenerationFlow,
       );
       break;
+    case 'nextjs':
+      await configureNextJsSourceMapsUpload(options, wizardOptions);
```

Comment:
don't know where else to put this: I think we need a call to `addSentryCliRc` for the nextjs source maps wizard flow

Reply:
Adressed by:
* printing the correct filename in the terminal when the "I have no CI" path is taken ([c7b87d9](https://github.com/getsentry/sentry-wizard/pull/372/commits/c7b87d9fced008bf97f3557b0f6c411673c8ac14))
* calling `addSentryCliRc` if users chose not to be redirected to the nextjs wizard ([6130b57](https://github.com/getsentry/sentry-wizard/pull/372/commits/6130b57f2c3547c159c550e5a4fe873e89f15635))